### ISSUE TITLE
[Bug]: TypeError: Cannot read properties of undefined (reading 'text') even in native @adminjs/logger component

### ISSUE THREAD

**Issue by ivictbor** [SoftwareBrothers/adminjs#1494]
### What happened?

Added @adminjs/logger to latest version of adminjs, log item is created fine but I can't see a diff:

![image](https://user-images.githubusercontent.com/1838656/236217093-c9cbadfc-0d89-4ec7-92be-8f26cb225567.png)

### Bug prevalence

Reproducible, always

### AdminJS dependencies version

```
npm ls --all | grep adminjs
├─┬ @adminjs/express@6.0.0
│ ├── adminjs@7.0.0 deduped
├─┬ @adminjs/import-export@3.0.0
│ ├── adminjs@7.0.0 deduped
├─┬ @adminjs/mongoose@4.0.0
│ ├── adminjs@7.0.0 deduped
├─┬ @adminjs/passwords@4.0.0
│ └── adminjs@7.0.0 deduped
├─┬ @adminjs/themes@1.0.0
│ ├── adminjs@7.0.0 deduped
├─┬ @devforth/adminjs-logger@5.0.1
│ ├── adminjs@7.0.0 deduped
├─┬ @devforth/adminjs-upload@4.0.4
│ └── adminjs@7.0.0 deduped
├─┬ adminjs@7.0.0
│ ├─┬ @adminjs/design-system@4.0.1
```

### What browsers do you see the problem on?

Chrome

### Relevant log output

```bash
ble.js:99 Uncaught TypeError: Cannot read properties of undefined (reading 'text')
    at styled.table.theme.theme (table.js:99:20)
    at _e (hoist-non-react-statics.cjs.js:103:38)
    at e.generateAndInjectStyles (hoist-non-react-statics.cjs.js:103:38)
    at e.generateAndInjectStyles (styled-components.browser.cjs.js:1:11336)
    at styled-components.browser.cjs.js:1:18616
    at styled-components.browser.cjs.js:1:18760
    at I (styled-components.browser.cjs.js:1:19206)
    at renderWithHooks (global.bundle.js:20564:19)
    at updateForwardRef (global.bundle.js:23480:21)
    at beginWork (global.bundle.js:25890:17)
s
```
```


### Relevant code that's giving you issues

Default code from the documentation

**Comment by ivictbor** [comment]
I was working fine before updating the versions. 
Since the error is in atoms/table I posted it here.
Was the theme interface changed? Theme colors gives me undefined on all components including old mine own. Am I missing something?

**Comment by dziraf** [comment]
@ivictbor can you try `@adminjs/logger` version 5.0.1?

**Comment by ivictbor** [comment]
@dziraf seams like it is 5.0.1 already:
![image](https://user-images.githubusercontent.com/1838656/236784857-267b5364-7056-43e8-acdb-acb831ef7952.png)

**Comment by dziraf** [comment]
`@devforthadminjs-logger` is not our package, I released `5.0.1` today.

**Comment by ivictbor** [comment]
> `@devforth/adminjs-logger` is not our package, I've released `5.0.1` today.

Oh, my bad, thanks, indeed, I forked to fix a bug and bumped patch (now I see it is a bad habit), will check now

**Comment by ivictbor** [comment]
@dziraf awesome, fixed, works fine now
